.github/scripts/seed-internal-testnet.sh

```diff
@@ -185,6 +185,23 @@ sleep $AVG_SECONDS_BETWEEN_BLOCKS
 updatedEvmUtilParams=$(curl https://api.app.internal.testnet.us-east.production.kava.io/kava/evmutil/v1beta1/params)
 printf "updated evm util module params\n %s" , "$updatedEvmUtilParams"
 
+# submit a kava token committee proposal
+COMMITTEE_PROP_TEMPLATE=$(
+  cat <<'END_HEREDOC'
+{
+    "@type": "/cosmos.gov.v1beta1.TextProposal",
+    "title": "The next big thing signaling proposal.",
+    "description": "The purpose of this proposal is to signal support/opposition to the next big thing"
+}
+END_HEREDOC
+)
+committeeProposalFileName="$(date +%s)-committee-proposal.json"
+touch $committeeProposalFileName
+echo "$COMMITTEE_PROP_TEMPLATE" >$committeeProposalFileName
+
+# committee 4 is the hard token committee
+kava tx committee submit-proposal 4 "$committeeProposalFileName" --gas 2000000 --gas-prices 0.01ukava --from god -y
```

Comment:
I like to use `--gas auto --gas-adjustment 1.5` for scripts like this unless I want to know about gas usage going over a certain threshold

Reply:
Also, maybe it's worth using a variable so we can put a name to the `4`?

Reply:
good point, this one shouldn't go over. I'll update it. and create a var for the token committee